resolwe_bio/tools/basespace_download.py

```diff
@@ -51,14 +72,16 @@ def main():
         headers = {'x-access-token': access_token}
 
         for file_id in file_ids:
-            file_name = get_file_name(session, file_id, headers)
-            download_file(session, file_id, file_name, headers)
-            output(args.output, 'filename={}'.format(file_name))
-    except Exception:
+            file_name, file_size = get_file_properties(session, file_id, headers)
+            download_file_repeatedly(args.tries, session, file_id, file_name, file_size, headers)
+            output(args.output, f'filename={file_name}')
+    except Exception as error:
+        print(str(error))
```

Comment:
Put this print in the `else` below, so you get traceback or this message, not both.